src/features/app/components/App.js

```diff
@@ -0,0 +1,30 @@
+import React, { Component } from 'react';
+
+import { Conference } from '../../conference';
+import config from '../../config';
+
+/**
+ * Main component encapsulating the entire application.
+ */
+export default class App extends Component {
+    /**
+     * Initializes the application title.
+     *
+     * @inheritdoc
+     */
+    componentWillMount() {
```

Comment:
I think we should not use componentWillMount(). It is being deprecated.
They have changed its name to [UNSAFE_componentWillMount()](https://reactjs.org/docs/react-component.html#unsafe_componentwillmount
) in latest React versions (with backwards compatibility)

https://reactjs.org/blog/2018/03/27/update-on-async-rendering.html

Reply:
Good point. I'll move it to the constructor instead, since in this case all we do is change the document.